organization/media/views.py

```diff
@@ -180,3 +181,30 @@ def get_context_data(self, **kwargs):
         context['json_event'] = json.dumps(events_data)
         return context
 
+
+class DynamicMultimediaView(Select2QuerySetSequenceView):
+    
+    paginate_by = settings.DAL_MAX_RESULTS
+
+    def get_queryset(self):
+
+        medias = Media.objects.all()
+        playlists = Playlist.objects.all()
+
+        if self.q:
+            medias = medias.filter(title__icontains=self.q)
+            playlists = playlists.filter(title__icontains=self.q)
+
+        qs = autocomplete.QuerySetSequence(medias, playlists, )
+
+        if self.q:
```

Comment:
second filter is not necessary

Reply:
@yomguy please see this commit below : 97e4e34
I will provide migration script for old playlists later. For now, I have to prioritize the templates for the moment. So I need this PR to be merge if it is ok for you.

Reply:
Sure ;)